dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/FATSuite.java

```diff
@@ -43,6 +48,6 @@
 })
 public class FATSuite {
     @ClassRule
-    public static RepeatTests r = RepeatTests.withoutModification().andWith(new FeatureReplacementAction().addFeature("jaxws-2.3").removeFeature("jaxws-2.2").removeFeature("jsp-2.2").withID("jaxws-2.3"));
+    public static RepeatTests r = RepeatTests.withoutModification().andWith(new FeatureReplacementAction().addFeature("jaxws-2.3").removeFeature("jaxws-2.2").removeFeature("jsp-2.2").removeFeature("servlet-3.1").removeFeature("jaxwstest-2.2").withID("jaxws-2.3"));
```

Comment:
Oh nice catch, I actually didn't intend to leave that removal here. That was a hold over from my initial attempt to keep both versions in one test. I actually hadn't thought of adding the jaxwstest-2.3 here, and maybe that would work? I guess it depends on when the config change described here actually occurs. The test is manually copying the test feature file `jaxwstest-2.3.mf` to the `wlp\lib` directory in the `@BeforeClass`, method so if the feature is there at that point, I suppose it would work.

Reply:
yeah I would expect you to add the 2.2 and 2.3 mf files during @BeforeClass and then remove them the same way you do now.